src/test/mir-opt/const_prop/usize_literal_propagation.rs

```diff
@@ -0,0 +1,10 @@
+// EMIT_MIR_FOR_EACH_BIT_WIDTH
```

Comment:
If you use `u32` (or some other platform independent size) instead of `usize`, you don't need this flag

Reply:
Indeed! Since we're not doing a `transmute` call in the test anymore, the `usize` should not be needed right? :)